packages/terra-abstract-modal/src/AbstractModal.jsx

```diff
@@ -31,6 +31,15 @@ const propTypes = {
    * If set to true, the modal will close when a mouseclick is triggered outside the modal.
    */
   closeOnOutsideClick: PropTypes.bool,
+  /**
+   * Element to fallback focus on if the FocusTrap can not find any focusable elements. Valid values are a valid
+   * dom selector string, a dom element, or a function that returns a dom element.
```

Comment:
We should specify what types of selectors are supported. I know xpath selectors are not supported in IE out-of-the-box.

Reply:
Are there concerns on what happens if selectors are not unique enough? Should this be limited to data attributes? I wonder if we should allow dom selector at all

Reply:
I'll document it as dom supported selectors. I don't want to get too involved with what browsers support which selectors, but I have no problem specifying a caution noting that you should ensure the browser selector is cross platform.

Reply:
I'd also like to keep the support, as it's the same prop types listed here. https://github.com/davidtheclark/focus-trap#usage

Reply:
Kept the dom selector with documentation, and the function. Had to drop the dom element